### ISSUE TITLE
Percussion should better preserve some timbre quality of original 

### ISSUE THREAD

**Issue by Cortexelus** [bmcfee/autochiptune#3]
The noise-based percussion isn't very interesting.
After listening to 20-25 remixes, I almost would rather take it out completely.
I want to hear transients.
I want to hear percussive diversity. 
I want something that preserves some timbre quality of the original.
What about MFCC nearest-neighbor from a sample bank of nes percussion? 
What about just bitcrushing the P?

**Comment by bmcfee** [comment]
> The noise-based percussion isn't very interesting.
> After listening to 20-25 remixes, I almost would rather take it out completely.
> I want to hear transients.
> I want to hear percussive diversity. 

Yeah, i agree with all of this.  WRT diversity, it should be much easier in librosa 0.4.1 via [this PR](https://github.com/bmcfee/librosa/pull/229), so you can compute onset envelopes in different frequency bands and synthesize them separately.

> I want something that preserves some timbre quality of the original.
> What about MFCC nearest-neighbor from a sample bank of nes percussion? 
> What about just bitcrushing the P?

PRs are always welcome!

**Comment by Cortexelus** [comment]
Bitcrushing the P

Cherry-picked example of it sounding good:
Before: https://soundcloud.com/genata_01/genko-ignatov
After: https://soundcloud.com/autochiptune/genata_01-genko-ignatov

However, it doesn't take the vox out of rap
https://soundcloud.com/autochiptune/thatnigganice-get-to-the-bread
